echo-pubsub/src/main/java/com/netflix/spinnaker/echo/pubsub/google/GooglePubsubSubscriber.java

```diff
@@ -143,17 +146,26 @@ public GooglePubsubMessageReceiver(Integer ackDeadlineSeconds,
     @Override
     public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
       String messagePayload = message.getData().toStringUtf8();
-      log.debug("Received message with payload: {}", messagePayload);
-
+      log.info("Received message with payload: {}", messagePayload);
       MessageDescription description = MessageDescription.builder()
           .subscriptionName(subscriptionName)
           .messagePayload(messagePayload)
           .pubsubSystem(pubsubSystem)
           .ackDeadlineMillis(5 * TimeUnit.SECONDS.toMillis(ackDeadlineSeconds)) // Set a high upper bound on message processing time.
           .retentionDeadlineMillis(TimeUnit.DAYS.toMillis(7)) // Expire key after max retention time, which is 7 days.
-          .artifacts(messageArtifactTranslator.parseArtifacts(messagePayload))
           .build();
       GoogleMessageAcknowledger acknowledger = new GoogleMessageAcknowledger(consumer);
+
+      try {
+        List<Artifact> artifacts = messageArtifactTranslator.parseArtifacts(messagePayload);
+        description.setArtifacts(artifacts);
+        log.info("artifacts {}", String.join(", ", artifacts.stream().map(Artifact::toString).collect(Collectors.toList())));
```

Comment:
FYI @jtk54 this should fix that bug where a malformed jinja template causes echo to retry processing a message forever